Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-slim-buster
```

Comment:
Is there a reason to use this image? I'd have thought you could use `3.9` or `3.9-slim`?

Reply:
Not particularly, ill have a look up what the 'buster' does. But general trying to keep the image small